src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -106,7 +109,7 @@ var createFiber = function(tag : number, key : null | string) : Fiber {
     memoizedInput: null,
     output: null,
 
-    hasPendingChanges: true,
+    pendingWorkPriority: 0,
```

Comment:
The fact that NoWork is a "higher priority" than anything else rather than lower, is kind of lame though. Requires double checks, but maybe those double checks are faster anyway due to comparison to null.

I'm not even sure any of this will remain anyway. Jordan had an idea of putting priorities in term of time stamps so that trees get automatically higher priority over time to avoid starvation.